From ae158de6ec463751ef4ac0a79b533f5658d84c9a Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Thu, 10 Mar 2016 18:25:45 -0500
Subject: [PATCH 10/34] ARM: dts: keystone: add qmss clock name

This patch adds the QMSS subsystem clock name
for Keystone platforms.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
---
 arch/arm/boot/dts/keystone-k2e-netcp.dtsi  | 1 +
 arch/arm/boot/dts/keystone-k2hk-netcp.dtsi | 1 +
 arch/arm/boot/dts/keystone-k2l-netcp.dtsi  | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/keystone-k2e-netcp.dtsi b/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
index a6827be..f695323 100644
--- a/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
+++ b/arch/arm/boot/dts/keystone-k2e-netcp.dtsi
@@ -19,6 +19,7 @@ qmss: qmss@2a40000 {
 	#address-cells = <1>;
 	#size-cells = <1>;
 	clocks = <&chipclk13>;
+	clock-names = "qmss_clk";
 	ranges;
 	queue-range	= <0 0x2000>;
 	linkram0	= <0x100000 0x4000>;
diff --git a/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi b/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
index 5bc984e..14cfbd8 100644
--- a/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
+++ b/arch/arm/boot/dts/keystone-k2hk-netcp.dtsi
@@ -19,6 +19,7 @@ qmss: qmss@2a40000 {
 	#address-cells = <1>;
 	#size-cells = <1>;
 	clocks = <&chipclk13>;
+	clock-names = "qmss_clk";
 	ranges;
 	queue-range	= <0 0x4000>;
 	linkram0	= <0x100000 0x8000>;
diff --git a/arch/arm/boot/dts/keystone-k2l-netcp.dtsi b/arch/arm/boot/dts/keystone-k2l-netcp.dtsi
index 6b3d552..b0a6cc2 100644
--- a/arch/arm/boot/dts/keystone-k2l-netcp.dtsi
+++ b/arch/arm/boot/dts/keystone-k2l-netcp.dtsi
@@ -19,6 +19,7 @@ qmss: qmss@2a40000 {
 	#address-cells = <1>;
 	#size-cells = <1>;
 	clocks = <&chipclk13>;
+	clock-names = "qmss_clk";
 	ranges;
 	queue-range	= <0 0x2000>;
 	linkram0	= <0x100000 0x4000>;
-- 
1.9.1

